packages/addon-dev/package.json

```diff
@@ -25,6 +25,14 @@
   "scripts": {
     "test": "jest"
   },
+  "peerDependencies": {
+    "rollup": "^4.6.0"
+  },
+  "peerDependenciesMeta": {
+    "rollup": {
+      "optional": true
```

Comment:
is this really true? 🤔 I mean there is no way that you can use this plugin in rollup without having rollup right?

Reply:
@mansona as far as I saw, all the plugins from https://github.com/rollup/plugins/tree/master/packages have this, so thought it would be best to follow their convention

Reply:
In theory maybe it's if someone build a plugin or wrapper on top of a plugin? IDK
But just left it their since I thought there must be a reason they have it

Reply:
I mean let's just go with what rollup does themselves 😂 👍